host/src/main/kotlin/host/transport/HostServer.kt

```
@@ -0,0 +1,93 @@
+        acceptThread.start()
+    }
+
+    override fun close() {
+        serverSocket.close()
+    }
+
+    private fun acceptLoop() {
+        while (!serverSocket.isClosed) {
+            val client = try {
+                serverSocket.accept()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)